src/tests/ftest/util/data_mover_test_base.py

```diff
@@ -644,7 +657,19 @@
         elif self.tool == "FS_COPY":
             self._set_fs_copy_params(src, dst)
         elif self.tool == "CONT_CLONE":
-            self._set_cont_clone_params(src, dst)
+            assert src_type in (None, "DAOS", "DAOS_UUID") # nosec
+            assert src_path is None # nosec
+            assert dst_type in (None, "DAOS", "DAOS_UUID") # nosec
+            assert dst_path is None # nosec
+            self._set_cont_clone_params(src_pool, src_cont,
```

Comment:
(pylint-too-many-function-args) Too many positional arguments for method call
(pylint-undefined-variable) Undefined variable 'src_pool'
(pylint-undefined-variable) Undefined variable 'src_cont'